controllers/userController.js

```diff
@@ -82,9 +85,10 @@ const userController = {
           lastName: req.body.lastName,
           email: req.body.email,
           password: bcrypt.hashSync(req.body.password, 10),
-        }).then((user) => {
+        }).then(async (user) => {
+          sendMail(user.email, template.subject, template.html);
           const response = {
-            message: 'Account created successfully',
+            message: 'Check your email spam box !',
```

Comment:
Podria ser tambien ✌️
```suggestion
            message: 'Account created successfully! Check your email spam box!',
```

Reply:
Si jefe!